Documentation/contributing/development/dev_setup.rst

```diff
@@ -656,15 +656,15 @@ These options enable logging functions in the datapath: ``cilium_dbg()``,
    monitor``.  In this case, ``bpftool prog tracelog`` can be used to retrieve
    debugging information from the eBPF based datapath. Both ``cilium_dbg()`` and
    ``printk()`` functions are available from the ``bpf/lib/dbg.h`` header file.
-   
+
 The image below shows the options that could be used as startup options by
 ``cilium-agent`` (see upper blue box) or could be changed at runtime by running
 ``cilium config <option(s)>`` for an already running agent (see lower blue box).
 Along with each option, there is one or more logging function associated with it:
 ``cilium_dbg()`` and ``printk()``, for ``DEBUG`` and ``cilium_dbg_lb()`` for
-``DEBUG_LB``. 
+``DEBUG_LB``.
 
-.. image:: _static/cilium-debug-datapath-options.svg 
+.. image:: _static/cilium-debug-datapath-options.svg
```

Comment:
While I appreciate the cleanup, those changes here are unrelated and should be in a separate commit (or even separate PR) to help with backporting conflicts and `git annotate` .